### DISCUSSION TITLE
Highstate Interval

### DISCUSSION THREAD

**Discussion by xaqpuyrthnbg** [Security-Onion-Solutions/securityonion#2081] [Unsupported Versions] [answered]
If you wanted to change how often Salt does a highstate, what files would you look to modify?

My impression was that the directory: /opt/so/saltstack/local/salt/salt contained these configs, but modifying these files with new values and manually running a hightstate does appear to work.  

Thanks in advance,

**Comment by m0duspwnens** [comment, marked as answer]
HI @thegodsquirrel . We currently have an issue (https://github.com/Security-Onion-Solutions/securityonion/issues/1144) that will change this process, but for now now we can change it another way.

If you copy the state `/opt/so/saltstack/default/salt/schedule.sls` to `/opt/so/saltstack/local/salt/schedule.sls` and modify the time within the local sls file, you can change the highstate schedule that way. Once we make the changes in the issue mentioned, you will want to remove the `/opt/so/saltstack/local/salt/schedule.sls` file and follow the new method.

**Reply by xaqpuyrthnbg** [reply]
Thank you @m0duspwnens ! I was so close.... 😑   I appreciate the fast response!
